### PR TITLE
Trigger a chef-client on the admin node after restoring chef data

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Comment by jsuchome** [comment]
Makes sense. +1

**Comment by dirkmueller** [comment]
+1

File: crowbar_framework/lib/crowbar/backup/restore.rb [hunk 1]
```diff
@@ -79,6 +79,9 @@ def restore_chef
           raise "Restore failed"
         end
 
+        Rails.logger.info("Re-runnig chef-client locally to apply changes from imported proposals")
```
Comment (MaximilianMeister) [inline crowbar_framework/lib/crowbar/backup/restore.rb]: typo runnig/running

Comment (vuntz) [inline crowbar_framework/lib/crowbar/backup/restore.rb]: https://github.com/crowbar/crowbar-core/pull/286

**Pull request merged**